### ISSUE TITLE
Invalid JSON data

### ISSUE THREAD

**Issue by AckslD** [OpenVPN/openvpn3-linux#47]
I'm trying to setup autoload. I made a folder `~/.config/openvpn3/autoload` containing two files `client.ovpn` and `client.autoload` where the latter is simply:
```
{
    "autostart": true,
}
```
When I run `openvpn3-autoload --directory .config/openvpn3/autoload` I get the error:
```
Invalid JSON data in ".config/openvpn3/autoload/client.ovpn": Expecting property name enclosed in double quotes: line 3 column 1 (char 25)
```

The `.ovpn` file shouldn't be json right? Or did I misunderstand something. Thanks in advance!

**Comment by ordex** [comment]
The JSON syntax is indeed wrong. If you have doubts, please check your file syntax on something like jsonlint.com

**Comment by AckslD** [comment]
Thanks for you reply @ordex. But that file shouldn't be a json file right?

**Comment by ordex** [comment]
The autoload config is expected to be a JSON file. (differently from the classic OpenVPN config which is *not* a JSON file)

**Comment by dsommers** [comment]
You have a trailing comma in the autoload config, which makes it invalid.

**Comment by AckslD** [comment]
Thanks! I got confused since it complained about the `client.ovpn` when the error was in fact in the `client.autoload`.

**Comment by dsommers** [comment]
Oh! That's a nice catch ... I'll see if I can fix the incorrect filename reference.  That is a confusing bug indeed; didn't realize that issue until your last comment.

**Comment by dsommers** [comment]
This is resolved by commit 7b643b2906ef1263899e334ae3dee5738d2977bc which is part of the v14_beta release.
